### DISCUSSION TITLE
.png wallpapers

### DISCUSSION THREAD

**Discussion by ghost** [t1m0thyj/WinDynamicDesktop#329] [Ideas]
I would love to see all wallpapers in .png format! .jpg has worse quality when applied as a wallpaper. I tried converting wallpapers files in WDD folder but it just replied with an error.

**Comment by t1m0thyj** [comment]
The default behavior of Windows for JPEG wallpapers that compresses them can be disabled to improve the image quality. It may help to use the script [here](https://github.com/t1m0thyj/WDD-scripts/tree/master/registry#disableenable-jpeg-compression) to disable JPEG compression.

An advantage of using PNG over JPEG for the wallpapers would be lossless compression. However there is also a disadvantage of it making the download size bigger. Users who live in a country with restricted Internet (e.g., China) have already complained about the downloads being slow as is, so I'm hesitant to make them even larger.
